libqtile/backend/wayland/core.py

```diff
@@ -492,7 +493,7 @@ def focus_window(self, win: window.WindowType, surface: Surface = None):
             win.surface.set_activated(True)
         self.seat.keyboard_notify_enter(surface, self.seat.keyboard)
 
-    def focus_by_click(self, event) -> None:
+    def _focus_by_click(self) -> None:
```

Comment:
Why the rename?

Reply:
It used to be called by `Qtile` but now it's only called by `Core`, so it's private.

Reply:
Yeah but then why not rename the X11 one? :D